src/services/transaction_reversal.rs

```diff
@@ -174,7 +174,7 @@ impl<'mpesa, Env: ApiEnvironment> TransactionReversalBuilder<'mpesa, Env> {
     /// Returns a `MpesaError` on failure.
     pub async fn send(self) -> MpesaResult<TransactionReversalResponse> {
         let url = format!(
-            "{}/reversal/v1/request",
+            "{}/mpesa/reversal/v1/request",
```

Comment:
Good catch!

Reply:
But the test still failing :/ due to invalid `ReceiverIdentifierType`, I used the default `11` from their site but didn't work.

Reply:
I think `"4"` should be the valid `ReceiverIdentifierType`, which should be `IdentifierTypes::ShortCode`

Reply:
I just realized you were not using the `IdentifierTypes` for the `identifier_type` parameter, I have added it with this suggestion https://github.com/collinsmuriuki/mpesa-rust/pull/44#discussion_r1027111429

Make sure you also update the tests

Reply:
Doesn't seem to work :/

Reply:
Still getting the same error?

Reply:
Yes, even on the saf API

Reply:
Could you share a snippet of the TransactionReversalBuilder's `send` method you are using to test this?

Reply:
```
pub async fn send(self) -> MpesaResult<TransactionReversalResponse> {
        let url = format!(
            "{}/mpesa/reversal/v1/request",
            self.client.environment().base_url()
        );

        let credentials = self.client.gen_security_credentials()?;

        let payload = TransactionReversalPayload {
            initiator: self.initiator,
            security_credentials: &credentials,
            command_id: self.command_id.unwrap_or(CommandId::TransactionReversal),
            transaction_id: self
                .transaction_id
                .ok_or(MpesaError::Message("transaction_id is required field"))?,
            receiver_party: self
                .receiver_party
                .ok_or(MpesaError::Message("receiver_party is required field"))?,
            receiver_identifier_type: self.receiver_identifier_type,
            result_url: self.result_url,
            timeout_url: self.timeout_url,
            remarks: self.remarks,
            occasion: self.occasion,
            amount: self
                .amount
                .ok_or(MpesaError::Message("amount is required field"))?,
        };

        let response = self
            .client
            .http_client
            .post(&url)
            .bearer_auth(self.client.auth().await?)
            .json(&payload)
            .send()
            .await?;

        if !response.status().is_success() {
            let value = response.json().await?;
            return Err(MpesaError::MpesaTransactionReversalError(value));
        };

        let response = response.json::<_>().await?;
        Ok(response)
    }
```